### PR TITLE
main: move src/main.py to __main__.py and fix pkg data

### PR DESCRIPTION
**Pull request by FFY00**
Signed-off-by: Filipe Laíns <lains@archlinux.org>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -46,6 +47,6 @@
     },
     test_requires=['pytest'],
     package_data={
-        'openapi': ['src/ratbag_emu/openapi/ratbag-emu.yaml'],
+        'ratbag_emu': ['openapi/ratbag-emu.yaml'],
```
Comment (bentiss) [inline setup.py]: this seems unrelated to the "move src/main.py to __main__.py"

It's best to have one change per commit

**Comment by FFY00** [comment]
Rebased this on #10.

File: ratbag_emu/__main__.py [hunk 2]
```diff
@@ -26,7 +27,7 @@ def main():
 
     # Run server
     server = connexion.FlaskApp(__name__,
-                                specification_dir='ratbag_emu/openapi/',
+                                specification_dir=pkg_resources.resource_filename('ratbag_emu', 'openapi/'),
```
Comment (bentiss) [inline ratbag_emu/__main__.py]: Is this change really related to the move from `ratbag-emu.py` to `__main__.py`? (hint: I have no ideas)

File: setup.py [hunk 3]
```diff
@@ -39,7 +39,6 @@
         'ratbag_emu.protocol',
         'ratbag_emu.protocol.devices',
         'ratbag_emu.protocol.util',
-        'ratbag_emu.server',
```
Comment (bentiss) [inline setup.py]: Looks like this one should be squashed into the previous commit though

File: ratbag_emu/__main__.py (hunk 2 above)
Comment (FFY00) [reply ratbag_emu/__main__.py]: Yes, we install the specification file as package data and `pkg_resources` allows us to fetch that file without going through hops finding the correct path.

Comment (bentiss) [reply ratbag_emu/__main__.py]: OK then
